build.gradle.kts

```diff
@@ -8,7 +8,7 @@ object Versions {
     const val avro = "1.10.0"
     const val brukernotifikasjonAvro = "1.2021.01.18-11.12-b9c8c40b98d1"
     const val isdialogmoteSchema = "1.0.5"
-    const val confluent = "5.5.0"
+    const val confluent = "6.1.3"
```

Comment:
Funker det i ved her etter denne endringen?

Reply:
Skal forsøke en deploy og teste litt, så får vi se.

Reply:
Fikk opprettet et møte uten problemer - er det en tilstrekkelig test? Så vidt jeg kan skjønne har det jo da blitt produsert noen meldinger på Kafka-topic for varsling.

Reply:
Ref: https://github.com/navikt/kafka-embedded-env/pull/18/files

Reply:
Ja burde fungerer fint da